cluster-setup-troubleshooting.md

```diff
@@ -0,0 +1,164 @@
+   ~~~
+
+   Errors at this stage potentially include:
+
+   - CPU incompatibility
+   - Other services running on port `26257` (CockroachDB's default). You can either stop those services or start your node with a different port, controlled with the [`--port` flag](start-a-node.html#flags).
+   - Networking issues that prevent the node from communicating with itself on its machine name. You can control the hostname CockroachDB uses with the [`--host` flag](start-a-node.html#flags).
+
+3. If the node appears to have started successfully, open a new terminal window, and attempt to execute a SQL statement:
+
+   ~~~ shell
+   $ cockroach sql --insecure -e "SHOW DATABASES"
+   ~~~   
+
+   You should receive a response that looks similar to this:
+
+   ~~~
+   +--------------------+
+   |      Database      |
+   +--------------------+
+   | crdb_internal      |
+   | information_schema |
+   | pg_catalog         |
+   | system             |
+   +--------------------+
+   ~~~
+
+   Errors at this stage potentially include: 
+
+   - `getsockopt: connection refused`, which indicates you have not included some flag that you used to start the node (such as `--port`).
+   - The `cockroach` node crashed; see if you can find the process using `ps`. In this case, file an issue.
+
+**Next step**: If you successfully completed these steps, try starting a multi-node cluster.
+
+## 2. Start a Multi-Node Cluster
+
+1. To scale your cluster, each node must be able to communicate with your first node. This might require that you quit your first node (`cockroach quit --insecure`) and then restart it specifying a `--host` that other nodes will be able to reach.
+
+2. If you're using a non-local environment, stop any running `cockroach` processes on the additional machines:
+
+   ~~~ shell
+   $ pkill cockroach
+   ~~~
+
+   {{site.data.alerts.callout_info}}If you're using these steps with a local deployment, this step will kill your first node.{{site.data.alerts.end}}
+
+3. On each machine, start the CockroachDB node, joining it to the first node:
+
+   ~~~ shell
+   $ cockroach start --insecure --logtostderr \
+   --store=testStore --join=[first node's host]
+   ~~~
+
+   Errors at this stage potentially include:
+
+   - [Networking issues](#networking-troubleshooting)
+   - [Nodes restarting existing clusters instead of joining first node](#node-wont-join-cluster)
+
+4. Visit the Admin UI on any node at `http://[node host]:8080`. All nodes in the cluster should be listed and have data rebalanced onto them.
+
+   Errors at this stage include:
+
+   - [Networking issues](#networking-troubleshooting)
+   - [Nodes not replicating data](#replication-error-in-a-multi-node-cluster)
+
+**Next step**: If you successfully completed these steps, try [securing your deployment](manual-deployment.html) (*troubleshooting docs for this forthcoming*) or reviewing our [support resources](support-resources.html).
+
+### Networking Troubleshooting
```

Comment:
There should probably be something in here about hostnames and DNS. By default, `cockroach` will advertise itself via hostname. If you are running in an environment without DNS or the hostname is not resolvable for some reason, the `cockroach` nodes won't be able to talk to each other. The `--advertise-host` flag can be used to work around the lack of DNS.